Sources/GDSCommon/UI/UIKit/SharedViews/Patterns/InstructionsWithImage/InstructionsWithImageViewController.swift

```diff
@@ -85,6 +97,27 @@ public final class InstructionsWithImageViewController: UIViewController {
         }
     }
     
+    /// secondaryButton button: ``SecondaryButton``. This is an optional property on the `viewModel`.
+    /// If it is `nil` on `viewModel` then the button is not displayed
+    @IBOutlet private var secondaryButton: SecondaryButton! {
+        didSet {
+            if let secondaryButtonViewModel = viewModel.secondaryButtonViewModel {
+                secondaryButton.isHidden = false
+                
+                if let icon = viewModel.secondaryButtonViewModel?.icon {
+                    secondaryButton.symbolPosition = icon.symbolPosition
+                    secondaryButton.icon = icon.iconName
+                }
+                
+                secondaryButton.setTitle(secondaryButtonViewModel.title,
+                                         for: .normal)
+                secondaryButton.accessibilityIdentifier = "secondaryButton"
```

Comment:
can the `accessibilityIdentifier` move outside of the if let? Might be helpful for testing.